stingray/powerspectrum.py

```diff
@@ -272,9 +279,11 @@ def rebin(self, df, method="mean"):
         """
 
         ## rebin power spectrum to new resolution
-        binfreq, binps, step_size = utils.rebin_data(self.freq[1:],
-                                                     self.ps[1:], df,
-                                                     method=method)
+        ## an empty variable is required to hold the yerr value returned
+        binfreq, binps, _, step_size = utils.rebin_data(self.freq[1:],
+                                                        self.ps[1:],
+                                                        self.ps_err[1:], df,
+                                                        method=method)
 
```

Comment:
Have you pulled the latest version of the code? I think with the new definition of the Fourier frequencies, the arguments should just be `self.freq`, `self.ps` and `self.ps_err`. Leaving out the first element of the array was only useful when the zeroth frequency was still part of the array.